odoo/addons/base/models/res_bank.py

```diff
@@ -126,7 +126,7 @@ def sanitize(arg):
                     value = [sanitize_account_number(i) for i in value]
                 else:
                     value = sanitize_account_number(value)
-                if 'like' in arg[1]:
+                if 'like' in arg[1] and value:
                     value = '%' + value + '%'
```

Comment:
you can remove this because `%` is added to the base

Reply:
@jva-odoo changes are done.